phase2/outline
# Adding Gitpod ready-to-code

## Is your feature request related to a problem? Please describe

Developers need to spend time setting up their environment to be able to start working on Outline.

## Describe the solution you'd like

Gitpod is a free service, with a preconfigured Outline dev environment.
It's ready to work in a few seconds with just 1 click.

## How to test this?
Click on this button,
[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/from-referrer/)

 (Authorize Gitpod through Github if this is the first time you use it)
When Gitpod interface finish loading, it will be after `yarn install` already ran (during prebuild), and in a few seconds you should see Storybook interface open without typing anything.
